common/static/common/js/karma.common.conf.js

```diff
@@ -0,0 +1,216 @@
+// Common configuration for Karma
+/* jshint node: true */
+'use strict';
```

Comment:
@andy-armstrong I thought about that first but then I realized it doesn't make any sense in common js modules. since karma is using common js modules the open var statements and functions won't go public.

http://stackoverflow.com/questions/23528616/common-practice-to-use-scoped-functions-for-commonjs-modules-or-unnecessary